scripts/benchmark.js

```diff
@@ -12,14 +12,28 @@ const babel  = require("babel-core");
 const zlib   = require("zlib");
 const fs     = require("fs");
 const path   = require("path");
+const commander = require("commander").Command;
```

Comment:
Because this was the only way I could see to fix my problem of the help message i.e. I needed to overwrite the script name and this is the only way I could do it I could see.